mozilla_django_oidc/auth.py

```diff
@@ -65,21 +67,30 @@ def create_user(self, claims):
 
     def verify_token(self, token, **kwargs):
         """Validate the token signature."""
-
+        nonce = kwargs.get('nonce')
         # Get JWT audience without signature verification
         audience = jwt.decode(token, verify=False)['aud']
 
         secret = self.OIDC_RP_CLIENT_SECRET
         if import_from_settings('OIDC_RP_CLIENT_SECRET_ENCODED', False):
             secret = base64.urlsafe_b64decode(self.OIDC_RP_CLIENT_SECRET)
 
-        return jwt.decode(token, secret,
-                          verify=import_from_settings('OIDC_VERIFY_JWT', True),
-                          audience=audience)
+        id_token = jwt.decode(token, secret,
+                              verify=import_from_settings('OIDC_VERIFY_JWT', True),
+                              audience=audience)
+
+        if import_from_settings('OIDC_USE_NONCE') and nonce != id_token['nonce']:
```

Comment:
Please add a default value here like you do in the rest of reference of ``OIDC_USE_NONCE``.